spec/round.rb

```diff
@@ -0,0 +1,55 @@
+class Round
+
+attr_reader :turns, :deck, :turn, :number_correct, :take_turn, :cards, :number_correct_by_category
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+
+  end
+
+  def current_card
+    deck.cards.first
+  end
+
+  def take_turn(guess)
+    turn = Turn.new(guess, current_card)
+      @turns << turn
+      if turn.correct?
+        @number_correct += 1
+      end
+      deck.cards.rotate!
+    return turn
+  end
```

Comment:
Several of the methods in this class are a little off for indentation. This method should look more like this, for example:
```ruby
  def take_turn(guess)
    turn = Turn.new(guess, current_card)
    @turns << turn
    if turn.correct?
      @number_correct += 1
    end
    deck.cards.rotate!
    turn
  end
```